src/styles/defaults.css

```diff
@@ -409,12 +409,19 @@ input:checked + label {
 
 /* DTable (grade distribution table */
 table td {
+    @apply my-2 text-base-content !important;
+  }
+  
+  table td.dtable-highlight {
+    @apply text-sm  bg-gray-200;
+  }
+/* table td {
```

Comment:
can you please remove the commented out css?  we're good to merge after that!

Reply:
Done!